.circleci/config.yml

```diff
@@ -166,90 +108,85 @@ jobs:
       - store_artifacts:
           path: integration-tests/screenshots
 
-  build_docker:
-    executor: deployer
+  security_audit:
+    executor: builder
     steps:
       - checkout
-      - attach_workspace:
-          at: ~/app
-      - setup_remote_docker:
-          docker_layer_caching: true
       - run:
-          name: Build container and publish to docker hub
-          command: |
-            APP_VERSION=$(egrep buildNumber build-info.json  | awk -F"\"" '{print $4}')
-
-            docker build --rm=false . \
-              --build-arg BUILD_NUMBER=$APP_VERSION \
-              --build-arg GIT_REF=$CIRCLE_SHA1 \
-              --tag "mojdigitalstudio/use-of-force:$APP_VERSION" \
-              --label "maintainer=andrew.lee@digital.justice.gov.uk" \
-              --label "app.version=$APP_VERSION" \
-              --label "build.version=$APP_VERSION" \
-              --label "build.number=$CIRCLE_BUILD_NUM" \
-              --label "build.url=$CIRCLE_BUILD_URL" \
-              --label "build.gitref=$CIRCLE_SHA1"
-
-            if [[ "${CIRCLE_BRANCH}" == "master" ]]; then
-              # publish to docker hub as the tag
-              docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
-              docker tag mojdigitalstudio/use-of-force:$APP_VERSION mojdigitalstudio/use-of-force:latest
-              docker push mojdigitalstudio/use-of-force:$APP_VERSION
-              docker push mojdigitalstudio/use-of-force:latest
-            fi
-  deploy_dev:
-    executor: deployer
-    steps:
-      - deploy_to_env:
-          env: 'dev'
-  deploy_preprod:
-    executor: deployer
-    steps:
-      - deploy_to_env:
-          env: 'preprod'
-  deploy_prod:
-    executor: deployer
-    steps:
-      - deploy_to_env:
-          env: 'prod'
-          slack_notification: true
+          name: update-npm
+          command: 'sudo npm install -g npm'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: install-npm
+          command: 'npm ci --no-audit'
+      - run:
+          name: run-security-audit
+          command: 'npm run security_audit'
 
 workflows:
   version: 2
   build-test-and-deploy:
     jobs:
       - build:
-          filters:
-            tags:
-              ignore: /.*/
+        filters:
+          tags:
+            ignore: /.*/
       - test:
           requires:
             - build
```

Comment:
I've also been renaming `build` and the `builder` executor to `validate` and `validator` since they don't actually do any building any more

Reply:
they kind of do in use of force - they build the compiled typescript which the integration tests are then run against

Reply:
interesting, didn't realise that :+1: